### PR TITLE
core(unused-javascript): increase threshold to 20KiB

### PR DESCRIPTION
**Pull request by connorjclark**
The threshold is too low.

Background: https://github.com/ampproject/amphtml/issues/28638

### CODE REVIEW

File: lighthouse-core/audits/byte-efficiency/unused-javascript.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ const UIStrings = {
 
 const str_ = i18n.createMessageInstanceIdFn(__filename, UIStrings);
 
-const IGNORE_THRESHOLD_IN_BYTES = 2048;
+const IGNORE_THRESHOLD_IN_BYTES = 20 * 1000;
```
Comment (patrickhulce) [inline lighthouse-core/audits/byte-efficiency/unused-javascript.js:23]: image

image
```suggestion
const IGNORE_THRESHOLD_IN_BYTES = 20 * 1024;
```


:)

Comment (connorjclark) [reply lighthouse-core/audits/byte-efficiency/unused-javascript.js:23]: lol oops

**Comment by connorjclark** [comment]
Smoke test only fails to run in CI ...

**Comment by stephengardner** [comment]
Cheering this one on 👏👏 thanks for the quick action @connorjclark

**Comment by patrickhulce** [comment]
Wow failed to produce a report at all that's really weird...just for a threshold change?

File: lighthouse-cli/test/smokehouse/test-definitions/byte-efficiency/byte-config.js [hunk 2]
```diff
@@ -34,6 +34,12 @@ const config = {
       'source-maps',
     ],
   }],
+  audits: [
+    {path: 'unused-javascript', options: {
```
Comment (brendankenny) [inline lighthouse-cli/test/smokehouse/test-definitions/byte-efficiency/byte-config.js:38]: error happens locally for me, too, either running `yarn smoke byte` or a regular run with `--config-path=lighthouse-cli/test/smokehouse/test-definitions/byte-efficiency/byte-config.js`, which gets the error message back: ``Runtime error encountered: Unable to locate audit: `unused-javascript`.``

This needs to be

```suggestion
    {path: 'byte-efficiency/unused-javascript', options: {
```

Comment (connorjclark) [reply lighthouse-cli/test/smokehouse/test-definitions/byte-efficiency/byte-config.js:38]: I  was running `yarn smoke perf` 🤦

**Comment by stephengardner** [comment]
Curious when this hits the live [site](https://developers.google.com/speed/pagespeed/insights/)?  Thanks!

**Comment by stephengardner** [comment]
Hi team, any final word on this? Thanks!

**Comment by connorjclark** [comment]
We have a scheduled release next week. Subscribe to the releases notification on this repo to be alerted

**Comment by stephengardner** [comment]
Has the release been delayed?  Is there a new estimated release date?
image
Thanks

**Comment by patrickhulce** [comment]
@stephengardner see #11025

**Comment by stephengardner** [comment]
> @stephengardner see #11025

Thanks @patrickhulce, following

**Comment by francescodicaprio** [comment]
Upvoting this. It reports as unused javascript even the GTAG for Google Analytics, which is well known as something that must be loaded in the head as Google itself suggests.

**Comment by connorjclark** [comment]
This has been fixed for two years now @francescodicaprio

**Comment by francescodicaprio** [comment]
My apologies, I thought merged stuff got closed, didn’t see the merged tag.
Anyway, it may exclude under 20k of stuff, but still reports as bad the gtag by Google. It’s funny seeing Google page speed reporting as bad a Google script that must be in the head.

**Pull request merged**